test/run-tests

```diff
@@ -468,7 +468,11 @@ def choose_task(gh, repos, images):
             commands = get_comment_commands(gh, owner, repo, number)
 
             for image in images:
-                status = statuses.get("linux-system-roles-test/" + image["name"])
+                status_context = "%s/%s" % (
+                    config.get("test_harness_name", "linux-system-roles-test"),
```

Comment:
Instead of defining default values at each place where the config is accessed, it makes more sense to have a dictionary with default values and then update those after parsing the config.json. Otherwise the default values need to be kept in sync (for this it should be at least a constant instead of the same string in several places).